src/situations/objets_trouves/modeles/store.js

```diff
@@ -16,7 +16,8 @@ export function creeStore () {
       appActive: null,
       appsVisitees: [],
       questionsFin: [],
-      consigneEcranAccueil: null
+      consigneEcranAccueil: null,
+      transitionFinTerminee: false
```

Comment:
Je m'attendais à l'utilisation d'une variable "etat" qui nous indique quel est l'état en cours de l'application object trouvés. etat = transitionFin, etat=questionDeLaFin, etat=fouillageTelephone…

Reply:
Mais c'est peut-être overkill, je ne sais pas.

Reply:
Hmm, je ne sais pas trop, sur le principe ça me plairait. La chose qui m'ennuie c'est qu'aujourd'hui on a cette notion d'état qui est au niveau de l'application (au sens large), à savoir `consigneTerminee`, `entrainementDemarre` etc..ce sont des états communs à toutes les situations. Ce que je comprend de ton commentaire c'est que tu souhaites faire la même chose au niveau de la situation `objets_trouves` et de son store.  Et du coup avoir deux systèmes d'état qui cohabitent (des états dans un états). Mes connaissances en framework JS sont un peu faible pour savoir si c'est une bonne pratique ou non, et si (comme disent nos amis anglo-saxons) "it plays nicely together".

Reply:
Par ailleurs , après avoir parcouru le repo, nous n'avons pas ce genre de comportement dans les autres situations. Ce qui bien entendu ne signifie pas que ça ne peut pas être fait :). Simplement on introduit une autre façon de faire.

Reply:
Je te propose qu'on en discute vendredi